app/code/Magento/InventoryInStorePickup/etc/di.xml

```diff
@@ -18,13 +18,9 @@
     <preference for="Magento\InventoryInStorePickupApi\Api\GetNearbySourcesByPostcodeInterface"
             type="Magento\InventoryInStorePickup\Model\GetNearbySourcesByPostcode"/>
     <preference for="Magento\InventoryInStorePickupApi\Api\Data\PickupLocationInterface" type="Magento\InventoryInStorePickup\Model\PickupLocation" />
-    <type name="Magento\InventoryInStorePickupApi\Model\GetNearbySourcesByPostcode">
-        <arguments>
-            <argument name="providers" xsi:type="array">
-                <item name="offline" xsi:type="object">Magento\InventoryInStorePickup\Model\DistanceProvider\Offline\GetNearbySourcesByPostcode</item>
-            </argument>
-        </arguments>
-    </type>
+    <preference for="Magento\InventoryInStorePickupApi\Api\GetNearbyPickupLocationsInterface" type="Magento\InventoryInStorePickup\Model\GetNearbyPickupLocations"/>
```

Comment:
The idea that previous implementation was limited by \Magento\InventoryDistanceBasedSourceSelection\Model\GetDistanceProviderCode.
But we already have implementation for each type of provider (offline and online) in `Magento_InventoryDistanceBasedSourceSelection`. 

So, instead of creation new bicycle to determination of lat and lng for center of circle, I have re-used `\Magento\InventoryDistanceBasedSourceSelectionApi\Api\GetDistanceInterface` instead.

If I misunderstood you, we can discuss this in offline.